android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -305,6 +326,10 @@ public static String getCloudDir() {
     return getResourceRoot() + KMDefault_UndefinedPackageID + File.separator;
   }
 
+  public static String getSVGDir() {
+    return getResourceRoot() + KMDefault_AssetSVG + File.separator;
+  }
```

Comment:
Why SVG? KMEA shouldn't know about SVG files. It should be a 'banner' folder if anything and should probably be a parameter passed into Engine rather than a hardcoded folder name

Reply:
An early iteration planned to just display an svg file (before we decided to use html blobs. I think we still need a known folder name for the assets to be copied in ll.852-860.

I can change the folder name from "svg" to "banner".

Reply:
We should make that a configurable path -- because the app dev may want to put keyman resources under an arbitrary subpath, e.g. 3rdparty/keyman/banner/ or something like that. Can do it in the same function as setting the HTML banner -- and if it is null/"" then there are no additional assets to copy.

Reply:
:+1: 
As long as keyboard developers keep their banner assets in a folder, it'll now get copied inside the KMManager.copyHTMLBannerAssets() call.